src/main/java/cn/nukkit/level/Sound.java

```diff
@@ -4,135 +4,288 @@
  * @author CreeperFace
  */
 public enum Sound {
```

Comment:
Review the changes, make backward compatibility values when possible and tag new sounds with the since annotation